config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :families
```

Comment:
Since you only implemented index, new, and create, I would restrict the routes here to be:
`resources :families, only: [:index, :new, :create]`

Reply:
👍